src/backend/distributed/transaction/backend_data.c

```diff
@@ -123,6 +124,7 @@ assign_distributed_transaction_id(PG_FUNCTION_ARGS)
 	MyBackendData->transactionId.transactionNumber = PG_GETARG_INT64(1);
 	MyBackendData->transactionId.timestamp = PG_GETARG_TIMESTAMPTZ(2);
 	MyBackendData->transactionId.transactionOriginator = false;
+	MyBackendData->citusInitiatedBackend = true;
```

Comment:
I don't think we need this here. We're already having a `LEFT JOIN` on the worker queries so we don't rely on `citusInitiatedBackend` set to true.

 And, that seems the reason you needed to set `transactionOriginator` to true in `MarkCitusInitiatedBackend ()`.

Our goal is to only mark coordinator processes with `citusInitiatedBackend`. Hence, the name `citusInitiatedBackend` sounds not accurate at this point. What about we use `citusInitiatedCoordinatorBackend` and do not set on `assign_distributed_transaction_id ()`? (If we do that, please go over the code and update all related things such as `MarkCitusInitiatedBackend ` -> `MarkCitusInitiatedCoordinatorBackend `)